script.js

```diff
@@ -1,4 +1,250 @@
+//global variables
+var gameMode = "rollDiceMode";
+var randomNumberOne = 0;
+var randomNumberTwo = 0;
+var playerNumber = 1;
+var myOutputValue = "";
+var combinedNumber = 0;
+var equals = false;
+var totalPlayerOneScore = 0;
+var totalPlayerTwoScore = 0;
+var currentPlayerOneScore = 0;
+var currentPlayerTwoScore = 0;
+var winningPlayer = "";
+
+//function to roll a dice
+var rollADice = function () {
+  var randomNumber = Math.floor(Math.random() * 6) + 1;
+  return randomNumber;
+};
+//function to format the string
+var formatString = function (
+  message,
+  input,
+  combinedNumber,
+  totalPlayerOneScore,
+  totalPlayerTwoScore
```

Comment:
`combinedNumber`, `totalPlayerOneScore` and `totalPlayerTwoScore` are already global variables, so it is not necessary to include them in the arguments of the helper function. This will help to reduce code when you call the function, and also reduce the possibility of missing out 1 argument (which will affect the logiv of the code) as well!